lib/fennec/udp/worker_supervisor.ex

```diff
@@ -4,14 +4,14 @@ defmodule Fennec.UDP.WorkerSupervisor do
 
   alias Fennec.UDP
 
-  def start_link(base_name) do
+  def start_link(opts, base_name) do
```

Comment:
That's true, but I wanted to match existing code style. But you're right, I should just change it globally.